docs/developer-guide/getting-started.md

```diff
@@ -67,7 +67,7 @@ FOCUS="no-auth-locations" make kind-e2e-test
 
 Valid values are defined in the describe definition of the e2e tests like [Default Backend](https://github.com/kubernetes/ingress-nginx/blob/master/test/e2e/defaultbackend/default_backend.go#L29)
 
-The complete list of tests can be found [here](e2e-tests.md)
+The complete list of tests can be found [here](../e2e-tests.md)
```

Comment:
This is not working. Have you ran this locally? Can you please check again the correct path, or maybe use the full path here?

Thanks :)

/lgtm cancel

Reply:
Yes, I've ran this locally. Here's what I get:

1. Run `make live-docs`, nothing of note/warnings in preamble:

  ```
    +] Building 1.2s (9/9) FINISHED
   ...
  
  Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
  INFO    -  Building documentation... 
  WARNING -  Config value: 'dev_addr'. Warning: The use of the IP address '0.0.0.0' suggests a production environment or the use of a proxy to connect to the MkDocs server. However, the MkDocs' server is intended for local development purposes only. Please use a third party production-ready server instead. 
  INFO    -  Cleaning site directory 
  INFO    -  Documentation built in 5.99 seconds 
  ```
2. Navigate to developer guide / getting started. Here's the output of `make live-docs`:

  ```
  [I 210605 06:00:40 handlers:135] Browser Connected: http://localhost:8000/developer-guide/getting-started/
  INFO    -  Browser Connected: http://localhost:8000/developer-guide/getting-started/
  ```

3. Click link on the word `here` in `The complete list of tests can be found here`. I see the correct page, and output of `make live-docs`:

  ```
  [I 210605 06:04:46 handlers:135] Browser Connected: http://localhost:8000/e2e-tests/
  INFO    -  Browser Connected: http://localhost:8000/e2e-tests/
  ```

We can also see that the previous link doesn't work and the relative link in this PR works because this is the directory listing, one `..` in `developer-guide/getting-started` brings us out and to a sibling of `developer-guide/`, and indeed `e2e-tests.md` and `developer-guide/` are both siblings under `docs/`:
- docs/
  - ...
  - e2e-tests.md
  - developer-guide/
    - getting-started.md

In contrast, on master branch, here's what I get:

1. Run `make live-docs`, note the warning in the preamble:

  ```
    ±  |master → origin {2} ?:3 ✗| → make live-docs
  [+] Building 2.8s (9/9) FINISHED
  ...
  
  Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
  INFO    -  Building documentation... 
  WARNING -  Config value: 'dev_addr'. Warning: The use of the IP address '0.0.0.0' suggests a production environment or the use of a proxy to connect to the MkDocs server. However, the MkDocs' server is intended for local development purposes only. Please use a third party production-ready server instead. 
  INFO    -  Cleaning site directory 
  WARNING -  Documentation file 'developer-guide/getting-started.md' contains a link to 'developer-guide/e2e-tests.md' which is not found in the documentation files. 
  INFO    -  Documentation built in 5.75 seconds
  ```

2. Navigate to developer guide / getting started. Here's the output of `make live-docs`:

  ```
  [I 210605 06:00:40 handlers:135] Browser Connected: http://localhost:8000/developer-guide/getting-started/
  INFO    -  Browser Connected: http://localhost:8000/developer-guide/getting-started/
  ```

3. Click link on the word `here` in `The complete list of tests can be found here`. I get a 404 page, and output of `make live-docs`:

  ```
  [W 210605 06:00:47 web:2239] 404 GET /developer-guide/getting-started/e2e-tests.md (172.17.0.1) 2.94ms
  WARNING -  404 GET /developer-guide/getting-started/e2e-tests.md (172.17.0.1) 2.94ms
  [I 210605 06:00:47 handlers:135] Browser Connected: http://localhost:8000/developer-guide/getting-started/e2e-tests.md
  INFO    -  Browser Connected: http://localhost:8000/developer-guide/getting-started/e2e-tests.md
  ```